web/src/main/java/se/inera/intyg/webcert/web/web/controller/testability/facade/util/CreateCertificateTestabilityUtil.java

```diff
@@ -235,11 +260,12 @@ private IntygUser getUser(String personId) {
     }
 
     private Patient getPatient(String patientId, String type, String typeVersion) {
```

Comment:
We get this error from this code if we try to use it ```java.lang.IllegalArgumentException: Unknown intygsTyp: fk7211```

Could we skip this step and instead just:

```java
final var patient = new Patient();
final var personnummer = Personnummer.createPersonnummer(patientId).orElseThrow();  
final var personFromPUService = patientDetailsResolver.getPersonFromPUService(
 personnummer
);
```

Reply:
There is some special logic for some legacy types that could be affected if we change this for all. Best would be to find a way to use what you suggest for any certificate types in CS, but the old way for the old types. 

You hardcode a solution for  now and we refactor the solution during next sprint when remove any references to FK7211 in Webcert. But as we have it for now in SupportedCertificateTypesUtil it is fine.